### PR TITLE
[CARBONDATA-848] Fixed error Presto Connector gives an error for select count(*) from table

### PR DESCRIPTION
**Pull request by bhavya411**
Verified the build for both Spark  2.1 and 1.6 
All Tests pass on both Spark 2.1 and 1.6
Manually Tested it on Presto

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1413/

File: pom.xml [hunk 1]
```diff
@@ -98,6 +98,7 @@
     <module>hadoop</module>
     <module>integration/spark-common</module>
     <module>integration/spark-common-test</module>
+    <module>integration/presto</module>
```
Comment (chenliang613) [inline pom.xml]: Add integration/presto module to parent pom.xml can be consider later once all issues be fixed and all test cases of presto be added.
So please remove it from your PR currently

**Comment by bhavya411** [comment]
@chenliang613  Removed presto module from pom.xml

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/1416/

**Comment by chenliang613** [comment]
verified. LGTM

**Comment by bhavya411** [comment]
Closing it manually

**Pull request closed without merging**